index.js

```diff
@@ -1,12 +1,28 @@
+const express = require('express');
+const mongoose = require('mongoose');
+const dbinitRoute = require('./src/routes/dbinitRoute');
 
-const express = require('express')
-const app = express()
-const port = 3000
+const app = express();
+const port = 3000;
+const dbUrl = 'mongodb+srv://admin:admin@cluster0.jg02e.mongodb.net/myFirstDatabase?retryWrites=true&w=majority';
```

Comment:
1. it's better to have a separated `config/db.js` file which stores this dbUrl variable 
2. it's better to name this variable `DB_URL` since it's a constant value as well as `PORT`